cockatrice/src/logger.cpp

```diff
@@ -51,16 +51,18 @@ void Logger::closeLogfileSession()
 void Logger::log(QtMsgType /* type */, const QMessageLogContext & /* ctx */, const QString &message)
 {
     logBuffer.append(message);
-    if(logBuffer.size() > LOGGER_MAX_ENTRIES)
-        logBuffer.removeFirst();
+    if (logBuffer.size() > LOGGER_MAX_ENTRIES)
```

Comment:
Size() => Returns the number of items in the vector.
Length() => Same as size() and count()
according to 5.8 docs